Lombiq.Tests.UI/Exceptions/HtmlValidationResultException.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Atata.HtmlValidation;
+using System;
+
+namespace Lombiq.Tests.UI.Exceptions
+{
+    public class HtmlValidationResultException : Exception
```

Comment:
The pair of this for accessibility is called `AccessibilityAssertionException` so I'd keep the naming pattern.